processor/spanprocessor/factory_test.go

```diff
@@ -59,11 +60,38 @@ func TestFactory_CreateTraceProcessor(t *testing.T) {
 // returns an error.
 func TestFactory_CreateTraceProcessor_InvalidConfig(t *testing.T) {
 	factory := &Factory{}
-	cfg := factory.CreateDefaultConfig()
 
-	tp, err := factory.CreateTraceProcessor(zap.NewNop(), exportertest.NewNopTraceExporter(), cfg)
-	require.Nil(t, tp)
-	assert.Equal(t, err, errMissingRequiredField)
+	testcases := []struct {
+		name string
+		cfg  Name
+		err  error
+	}{
+		{
+			name: "missing config",
```

Comment:
:) I keep adding underscores to those because then I can search for them directly on the source, cut & paste from terminal...

Reply:
Good point!